int16 balancestate
int16 modID
time stamp
